### PR TITLE
BDM-124 quote calculation for policy

### PR DESCRIPTION
**Pull request by Arvolear**
Despite the function and tests implementation, there were some problems in dependencies and linter that are fixed now.

### CODE REVIEW

File: contracts/PolicyBook.sol [hunk 1]
```diff
@@ -158,4 +154,56 @@ contract PolicyBook is IPolicyBook, ERC20 {
...
+      (PRECISION * (_utilizationRatioPercentage - RISKY_ASSET_THRESHOLD_PERCENTAGE)) /
+        (PERCENTAGE_100 - RISKY_ASSET_THRESHOLD_PERCENTAGE);
+
+    return
+      MAXIMUM_COST_NOT_RISKY_PERCENTAGE +
+      (riskyRelation * (MAXIMUM_COST_100_UTILIZATION_PERCENTAGE - MAXIMUM_COST_NOT_RISKY_PERCENTAGE)) /
+      PRECISION;
+  }
+
+  function getQuote(uint256 _durationDays, uint256 _tokens) external view override returns (uint256 _daiTokens) {
```
Comment (armenArakelian) [inline contracts/PolicyBook.sol:185]: according to the doc right now max amount of days is 365 and it is always better to keep everything in contract is seconds not days

Comment (Arvolear) [reply contracts/PolicyBook.sol:185]: This is a provided function signature. We'd better discuss it with Applicature guys.

File: contracts/PolicyBook.sol [hunk 2]
```diff
@@ -158,4 +152,55 @@ contract PolicyBook is IPolicyBook, ERC20 {
...
+
+  uint256 public constant MINIMUM_COST_PERCENTAGE = 5 * PRECISION;
+  uint256 public constant RISKY_ASSET_THRESHOLD_PERCENTAGE = 70 * PRECISION;
+  uint256 public constant MAXIMUM_COST_NOT_RISKY_PERCENTAGE = 30 * PRECISION;
+  uint256 public constant MAXIMUM_COST_100_UTILIZATION_PERCENTAGE = 150 * PRECISION;
+
+  uint256 public daiInThePoolTotal;
+  uint256 public daiInThePoolBought;
+
+  function calculateWhenNotRisky(uint256 _utilizationRatioPercentage) private pure returns (uint256) {
```
Comment (applicature-yaroslav-k) [inline contracts/PolicyBook.sol:168]: Isn't it's better to use safe math for calculations below?

Comment (Arvolear) [reply contracts/PolicyBook.sol:168]: It definitely is, but in order to overflow those uint256 one will need to have billions of quadrillions DAIs. (Which is quite impossible). However for safety reasons it makes sense.

File: contracts/PolicyBook.sol (hunk 1 above)
Comment (armenArakelian) [reply contracts/PolicyBook.sol:191]: I have discussed it, we can keep in seconds, it is fine for now but probably will change it in the later version

**Pull request merged**